src/containers/Welcome/Step4/Step4.tsx

```diff
@@ -215,7 +224,7 @@ const Step4 = (p: Wizard.StepProps) => {
               id="Step2-PolicyTerms"
               label={(
                 <Trans tOptions={{ lng: getCurrentCountryCheckbox(currentCountry) }} i18nKey="consent:agree">
-                  I have read, understood, and agree to the terms of the Virufy Privacy Policy.
+                  I have read, understood, and agree to the terms of the <LinkPurple to={privacyPolicy} target="_blank">Virufy Privacy Policy</LinkPurple>.
```

Comment:
Is not necessary to store privacy policy in a const, please just put the link into `to` prop, like this:
`to="https://virufy.org/privacy_policy"`